app/boards/arm/nice_nano/nice_nano.dtsi

```diff
@@ -71,7 +70,7 @@
 &flash0 {
     /*
      * For more information, see:
-     * http://docs.zephyrproject.org/latest/devices/dts/flash_partitions.html
+     * http: //docs.zephyrproject.org/latest/devices/dts/flash_partitions.html
```

Comment:
Looks like you inadvertently broke this URL.

Reply:
Indeed. Hate when my auto formatting butchers that. Good catch. Thanks!